src/vtbackend/primitives.h

```diff
@@ -18,9 +15,13 @@
 // - [ ] rename all History to Scrollback
 // - [ ] make sense out of all the semantically different line primitives.
 
-namespace terminal
+namespace vtbackend
 {
 
+using LineCount = vtpty::LineCount;
```

Comment:
I would expect it to be inside vtbackend itself

Reply:
LineCount and ColumnCount along with PageSize are in vtpty, because a PTY exposes and configures the page size. this is why we're having it there